nengo/neurons.py

```diff
@@ -89,6 +89,38 @@ def gain_bias(self, max_rates, intercepts):
 
         return gain, bias
 
+    def max_rates_intercepts(self, gain, bias):
+        """Compute the max_rates and intercepts corresponding to the given
+        gain and bias.
+
+        Note that this default implementation is very slow! Whenever possible,
+        subclasses should override this with a neuron-specific implementation.
+
+        Parameters
+        ----------
+        gain : ndarray(dtype=float64)
+            Gain associated with each neuron. Sometimes denoted alpha.
+        bias : ndarray(dtype=float64)
+            Bias current associated with each neuron.
+
+        Returns
+        -------
+        max_rates : ndarray(dtype=float64)
+            Maximum firing rates of neurons.
```

Comment:
Yeah I would say the place to document what we mean by "max_rate" is in the `Ensemble.max_rates` docstring (which does contain a good deal more information).  I could copy that docstring to here as well though.